configure.ac

```
@@ -3974,6 +3974,82 @@ AS_HELP_STRING([--disable-crypto-auth],[Disable cryptographic authentication]),
        AC_MSG_RESULT(yes)
 )
 
+dnl ************************************************************
+dnl disable basic authentication
+dnl
+AC_MSG_CHECKING([whether to enable basic authentication method])
+AC_ARG_ENABLE(basic-auth,
+AS_HELP_STRING([--enable-basic-auth],[Enable basic authentication])
```

```suggestion
AS_HELP_STRING([--enable-basic-auth],[Enable basic authentication (default)])
```

Indicating which one is the default would be useful IMO. Same for the rest of the new settings.

---

Existing options could also benefit greatly if their defaults would be known. It's also difficult to find it out after the fact. It'd be a good opportunity to start adding them with newly added options. The less work it will require to retrofit the missing ones later.

[IMO]